src/codeu/chat/common/Access.java

```diff
@@ -0,0 +1,22 @@
+package codeu.chat.common;
+
+public enum Access {
```

Comment:
This doesn't conform to the google style guide: https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes

I'm skeptical that you need the extra methods here too. I know java lets you do a lot with enum classes, but usually you just need a few definitions and that's it.